source/manual/set-up-aws-account.html.md

```diff
@@ -0,0 +1,67 @@
+---
+owner_slack: "#govuk-2ndline"
+title: Set up your AWS account
+section: AWS accounts
+layout: manual_layout
+parent: "/manual.html"
+last_reviewed_on: 2018-09-03
+review_in: 6 months
+redirects:
+  - /manual/user-management-in-aws.html
+---
+
+To work with [govuk-aws](https://github.com/alphagov/govuk-aws) and [govuk-aws-data](https://github.com/alphagov/govuk-aws-data),
+you will require an account in AWS.
+
+## 1. Request a GDS AWS account
+
+GDS maintains a central account for AWS access. You will need to request an account from the Technology and Operations team.
+
+👉 [Request an account](https://gds-request-an-aws-account.cloudapps.digital)
+
+## 2. Sign in to AWS
+
+To sign in, go to [the GDS AWS Sign page](https://gds-users.signin.aws.amazon.com/console), and use the following credentials:
+
+- "Account ID or alias": `gds-users`
+- Username: your `@digital.cabinet-office.gov.uk` email address
+- Password: your password
+
+👉 [Sign in to AWS GDS account](https://gds-users.signin.aws.amazon.com/console)
+
+## 3. Set up your MFA
+
+You have to set up [Multi-Factor Authentication (MFA)][MFA].
+
+## 4. Create access keys
+
+Create a pair of [access keys][access-keys].
+
+## 5. Get the appropriate access
+
+An account in AWS doesn't give you access to anything, you'll need to be given rights.
+
+Add yourself to the lists of users found in [the data for the infra-security project][infra-terra].
+
+The identifier you need add is called the "User ARN". You can find this by going
+to the [users page in AWS IAM][iam], and find your ***** profile.
```

Comment:
Is there meant to be a word here?

Reply:
Hah, yeah I think so.

Reply:
I think it works if we simply remove the asterisks.

Reply:
What about
```
You can find this by going to the [users page in AWS IAM][iam], and selecting your profile.
```
?

Reply:
Fixed, cheers.